e-commerce/pom.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.cgi.bonnie</groupId>
+        <artifactId>bonnie-parent</artifactId>
+        <version>1.0-SNAPSHOT</version>
```

Comment:
These repeated `1.0.SNAPSHOT `-s are bothering me. I know we have them already all over the place. I'm contemplating to extract them into a spring property.